app/views/todos/todo_home.erb

```diff
@@ -0,0 +1,31 @@
+
+
+<a href="/logins/login_delete">Logout</a><br>
+
+
+
+Welcome to the ToDo List!
+
+Get started now.
+
+<!-- The ToDo list will be a list of Array objects. Use an each do loop to show the items on the page. Todo "completed" should just be a check box.
+ -->
+
+<h2>ToDos</h2>
+
+  <p><% @todos.each do |todo| %></p><br>
+
+    <p><%= todo.completed %></p> <p><%= todo.title %></p> <p><%= todo.(session[:user_id] %></p><br>
```

Comment:
What is your goal with `<%= todo.(session[:user_id] %>`? That is, what do you expect will happen if that works as it does in your mind?